request.go

```diff
@@ -537,6 +538,8 @@ type Request interface {
 	Code() int32
 	// Body fills an encoder with a request body.
 	Body(resolver SchemaResolver, enc *msgpack.Encoder) error
+	// Context returns a context of the request.
+	Context() context.Context
```

Comment:
It's time for the important conversation: about naming.

I propose to rename `Context()` -> `Ctx()` in the interface and `WithContext()` -> `Context()` for setters. The reason: all our setter have the same name-pattern: Arg(arg). It would be obvious the same for contexts.

Reply:
I am ok to change it, but you have already [proposed](https://github.com/tarantool/go-tarantool/pull/173#discussion_r906672223) another naming a bit earlier, which one will be better?

Reply:
Fixed.